bitnami/redis/templates/scripts-configmap.yaml

```diff
@@ -331,7 +331,7 @@ data:
             sentinel_info_command="{{- if and .Values.auth.enabled .Values.auth.sentinel }}REDISCLI_AUTH="\$REDIS_PASSWORD" {{ end }}timeout {{ .Values.sentinel.getMasterTimeout }} redis-cli -h $REDIS_SERVICE -p $SENTINEL_SERVICE_PORT sentinel get-master-addr-by-name {{ .Values.sentinel.masterSet }}"
         fi
         info "about to run the command: $sentinel_info_command"
-        retry_while "eval $sentinel_info_command" 2 1
+        retry_while "eval $sentinel_info_command" 10 1
```

Comment:
Should we still retry twice, but sleep longer between retries (maybe 5s will avoid the race with greater probability?)